knime_extension/src/nodes/spatialtool.py

```diff
@@ -1195,6 +1195,72 @@ def execute(self, exec_context: knext.ExecutionContext, input_table):
         return knut.to_table(grid, exec_context)
 
 
+############################################
+# Create H3 Grid Node
+############################################
+
+
+@knext.node(
+    name="Create H3 Grid",
+    node_type=knext.NodeType.MANIPULATOR,
+    icon_path=__NODE_ICON_PATH + "CreateH3Grid.png",
+    category=__category,
+    after="",
+)
+@knext.input_table(
+    name="Input Table",
+    description="Input table of Create H3 Grid, should be a polygon with WGS-84/EPSG:4326 projection",
+)
+@knext.output_table(
+    name="Output Table",
+    description="Output table of Created H3 Grid",
+)
+class CreateH3Grid:
+    """Create H3 Grid node.
+    This node creates hexagonal grid inside the input polygon using [H3](https://h3geo.org/).
```

Comment:
Mention that the node will always project into epsg:4326 and use unary_union to create a single polygon for a multiple row input table